src/Nethermind/Nethermind.Shutter/ShutterCrypto.cs

```diff
@@ -217,8 +217,8 @@ public static bool CheckSlotDecryptionIdentitiesSignature(
     }
 
     [SkipLocalsInit]
```

Comment:
Hm is this even doing anything when no arrays are allocated?

Reply:
Good catch, it's not needed here